hugegraph-it/hg-it-minicluster/src/main/java/org/apache/hugegraph/it/config/ClusterConfImpl.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hugegraph.it.config;
+
+public class ClusterConfImpl implements ClusterConf {
```

Comment:
not sure why this class is empty

Reply:
ClusterConfImpl is designed to reserve the configuration of nodes, which is used in the production of configuration files e.g. xml files. It will be completed in the next push